dspace-spring-rest/src/test/java/org/dspace/app/rest/MetadatafieldRestRepositoryIT.java

```diff
@@ -124,4 +163,190 @@ public void findByNullSchema() throws Exception {
+                            .param("schemaId", metadataSchema.getID() + "")
+                            .content(new ObjectMapper().writeValueAsBytes(metadataFieldRest))
+                            .contentType(contentType))
+                    .andExpect(status().isCreated())
+                    .andDo(result -> idRef.set(read(result.getResponse().getContentAsString(), "$.id")));
+
+            MetadataField metadataField = metadataFieldService.find(context, idRef.get());
+            assertThat(metadataField, notNullValue());
+
+            assertEquals(metadataField.getMetadataSchema(), metadataSchema);
+            assertEquals(metadataField.getElement(), ELEMENT);
+            assertEquals(metadataField.getQualifier(), QUALIFIER);
+            assertEquals(metadataField.getScopeNote(), SCOPE_NOTE);
+
+        } finally {
+            deleteMetadataFieldIfExists();
+        }
+    }
+
+    @Test
+    public void createUnauthauthorized() throws Exception {
+
+        MetadataFieldRest metadataFieldRest = new MetadataFieldRest();
+        metadataFieldRest.setElement(ELEMENT);
+        metadataFieldRest.setQualifier(QUALIFIER);
+        metadataFieldRest.setScopeNote(SCOPE_NOTE);
+
+        try {
+            getClient()
+                    .perform(post("/api/core/metadatafields")
+                            .param("schemaId", metadataSchema.getID() + "")
+                            .content(new ObjectMapper().writeValueAsBytes(metadataFieldRest))
+                            .contentType(contentType))
+                    .andExpect(status().isUnauthorized());
+        } finally {
+            deleteMetadataFieldIfExists();
+        }
+    }
+
+    @Test
+    public void deleteSuccess() throws Exception {
+
+        MetadataField metadataField = createMetadataField();
+
+        try {
+
+            assertThat(metadataFieldService.find(context, metadataField.getID()), notNullValue());
+
+            getClient(getAuthToken(admin.getEmail(), password))
+                    .perform(delete("/api/core/metadatafields/" + metadataField.getID()))
+                    .andExpect(status().isNoContent());
+
+            assertThat(metadataFieldService.find(context, metadataField.getID()), nullValue());
+
+        } finally {
+            deleteMetadataFieldIfExists();
+        }
+    }
+
+    @Test
+    public void deleteUnauthorized() throws Exception {
+
+        MetadataField metadataField = createMetadataField();
+
+        try {
+
+            assertThat(metadataFieldService.find(context, metadataField.getID()), notNullValue());
+
+            getClient()
+                    .perform(delete("/api/core/metadatafields/" + metadataField.getID()))
+                    .andExpect(status().isUnauthorized());
+
+            assertThat(metadataFieldService.find(context, metadataField.getID()), notNullValue());
+
+        } finally {
+            deleteMetadataFieldIfExists();
+        }
+    }
+
```

Comment:
add a test for the forbidden scenario (loggedin user without  admin rights)

Reply:
This test was added. See `deleteUnauthorizedEPersonNoAdminRights()` in the code